app/controllers/base_public_controller.rb

```diff
@@ -2,13 +2,14 @@ class BasePublicController < ApplicationController
   include ClaimSessionTimeout
 
   helper_method :current_policy, :current_policy_routing_name, :claim_timeout_in_minutes
+  before_action :add_view_paths
   before_action :end_expired_claim_sessions
   after_action :update_last_seen_at
 
   private
 
   def current_policy
-    Policies[current_policy_routing_name]
+    Journeys.policy_for_routing_name(current_policy_routing_name)
```

Comment:
```suggestion
    Journeys.policies_for_routing_name(journey_name)
```

Reply:
Will do this when we incorporate `current_journey`